comms/storage/transcode/jobs.go

```diff
@@ -479,6 +477,12 @@ func (jobman *JobsManager) startWorker(workerNumber int) {
 			continue
 		}
 
+		// Only process pending jobs
+		if job.Status != JobStatusPending {
+			msg.Ack()
+			continue
+		}
```

Comment:
My concern here is based on a lack of understanding about how KV messages will work with QueueSubscribe...

* worker1 pulls a job... updates it to "in progress"
* worker1 crashes... job is stuck in "in progress"
* NATS never gets an ACK for msg from worker1 and it times out.
* A different QueueSubscribe worker picks up the baton... but what does the `job` record look like?  Is it the initial "pending" version?  Or is it the stuck "in progress" version.

I think that NATS will probably actually re-deliver the initial message... but if you were to do a vanilla kv get... you'd see "in progress" version.

So... will probably work, let's test it in prod! 😆

Reply:
Yes I observed that an ffmpeg job can take longer than the timeout window... and that NATS would give to a second worker while the first one was still running...

BUT this line takes care of it:
https://github.com/AudiusProject/audius-protocol/blob/main/comms/storage/transcode/jobs.go#L414-L415

Reply:
ohhh nice!